types/trusted-types/trusted-types-tests.ts

```diff
@@ -1,9 +1,6 @@
-import TT from 'trusted-types';
+import { trustedTypes, TrustedTypesEnforcer, TrustedTypeConfig } from 'trusted-types';
 
-// $ExpectType TrustedTypePolicyFactory
-TT;
-
-// $ExpectError
+// $ExpectType InternalTrustedTypePolicyFactory
 trustedTypes;
 
 // $ExpectType TrustedTypePolicyFactory | undefined
```

Comment:
Please make this `const globalTrustedTypes = window.trustedTypes` or use `window.trustedTypes` everywhere below.

Reply:
Let me isolate it into two separate files. One for testing the browser trusted types, the other one for testing the module.